pom.xml

```diff
@@ -1725,7 +1725,9 @@
       yyyy-MM-dd'T'HH:mm
     </maven.build.timestamp.format>
     <buildDate>${maven.build.timestamp}</buildDate>
+    <!-- these two control the Java language version and JVM runtime version supported -->
     <compileSource>1.8</compileSource>
```

Comment:
I believe that compile source and target allow setting `8` in addition to `1.8` while release requires `8` - can we standardize on a single value for all of these instead of have both 1.8 and 8 around?

Reply:
Ah, indeed. https://docs.oracle.com/en/java/javase/11/tools/javac.html